### ISSUE TITLE
Conflict when using with colorize

### ISSUE THREAD

**Issue by wendy0402** [mattheworiordan/capybara-screenshot#115]
I'm using colorize in my project.

```
/Users/my_user/.rvm/gems/ruby-2.1.5/gems/colored-1.2/lib/colored.rb:70:in `colorize': no implicit conversion of Hash into String (TypeError)
	from /Users/my_user/.rvm/gems/ruby-2.1.5/gems/colorize-0.7.5/lib/colorize/class_methods.rb:75:in `block (2 levels) in color_methods'
	from /Users/my_user/.rvm/gems/ruby-2.1.5/gems/capybara-screenshot-1.0.4/lib/capybara-screenshot/rspec/text_reporter.rb:28:in `output_screenshot_info'
	from /Users/my_user/.rvm/gems/ruby-2.1.5/gems/capybara-screenshot-1.0.4/lib/capybara-screenshot/rspec/text_reporter.rb:22:in `example_failed_with_screenshot'
	from /Users/my_user/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/reporter.rb:134:in `block in notify'
	from /Users/my_user/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/reporter.rb:133:in `each'
	from /Users/my_user/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/reporter.rb:133:in `notify'
	from /Users/my_user/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/reporter.rb:95:in `example_failed'
	from /Users/my_user/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/example.rb:348:in `finish'
	from /Users/my_user/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/example.rb:300:in `fail_with_exception'
	from /Users/my_user/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/example_group.rb:460:in `block in run'
	from /Users/my_user/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/example_group.rb:498:in `each'
	from /Users/my_user/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/example_group.rb:498:in `for_filtered_examples'
	from /Users/my_user/.rvm/gems/ruby-2.1.5/gems/rspec-core-3.1.7/lib/rspec/core/example_group.rb:502:in `block in for_filtered_examples'`
```

**Comment by mattheworiordan** [comment]
See https://github.com/mattheworiordan/capybara-screenshot/pull/111, we should have a fix for this soon.

**Comment by mattheworiordan** [comment]
This is now fixed with #118, please get the latest version
